### DISCUSSION TITLE
Propositions

### DISCUSSION THREAD

**Discussion by mbab** [DivinumOfficium/divinum-officium#3262] [Ideas]
Here are some propositions for changes to project, I would like to hear Your opinion about:

1. **Monastic directories:**

Those directories: SanctiM TemporaM CommuneM, can be removed with files moved to _regular_ directory with name like 07-11m.txt, most of them will be unnecessary as they contains only links to regular.
This should speed up generating Monastic ordo

2. **Local (language based) calendars:**

- can be implemented as overlay (consist only changes) to global one

part of example Tabulae/Kalendaria/Polski/1954.txt

05-03=05-03_pl=Beatæ Mariæ Virginis Reginae atque Patronae Principalis Poloniae=6=
05-04=05-03=Inventione Sanctae crucis=5=Ss. Martyrum=1=
05-07=07-07_pl=S. Floriani, Martyris=3
05-08=05-08_pl=S. Stanislai, Episcopi Et Martyris, Patroni Principalis Polonia=6=

- perhaps there should be option if choosing language should also choose local calendar (what should be default?)
- for English it can introduce new language options as English-US English-UK ...

nb. Tabulae should move 2 directories up (to web/www)

3. **Regula:**

- put Regula in one file with verse numbering, use regulaheads to select verses, this should be easier for vernaculars to follow
(and for Cistercian :-)

**+**

**Comment by fiapps** [comment]
Proposal 1. sounds like a good idea. The only issue here is that these currently independent files may contain different text: one copy of the Latin may be better proofread or have more accents, or the vernacular translations may vary. When merging these, it is important to save the better of the two.

I am indifferent to proposal 3. My only concern is that it might be slower for the program to parse the entire Rule to print the day's selection.

Regarding proposal 2, I like the idea of overlays, but I don't think they should be tied to languages. Proper calendars are for regions like countries or dioceses, or for a religious Order, Congregation, priestly Society, etc. So it should not be English-UK, but England_and_Wales, or England_and_Wales-Plymouth, or OFM, FSSP, …

**Comment by FAJ-Munich** [comment]
Regarding Proposal 3, I think it should be a third item to choose from so that one can combine certain rubrics with certain language translations following either a local or the general calendar. As a fourth option, imho, one could also combine a Switch Roman vs. Monastic such that one actually has a choice like:

Roman Rubrics: Roman 1570, Roman 1910 (or 1888?), Roman DA (1954), 1955, 1960, OP, Newcal
Monastic Rubrics: Benedictine 1617 (PDF available online), 1910? (no source?), DA (1930, available online), 1963 (already implemented) 

In addition, the layover would be necessary to be well maintained also regarding the transfers and the commemorations. The latter being a general issue especially under 1910 but also DA rubrics at the moment. For the Monastic 1930 rubrics, in addition, one would need to implement dynamic readings to the days within common octaves like Immaculate Conception, which could be addressed by a general makeover of the commemorations mechanism or potentially (ab)using the meachnism in place for Scripture of the months of August to November.
